packages/flutter_tools/lib/src/localizations/gen_l10n_templates.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+const String emptyPubspecTemplate = '''name: synthetic_package
```

Comment:
I think this also needs to have an SDK constraint too? I'm not sure if build will prefer the one in the package_config.

Reply:
The `build_runner` exception no longer appears with a pubspec that does not specify an SDK constraint. Could you clarify what you mean by build preferring the one in package_config over providing one here?

Reply:
If it doesn't happen at all, then I would leave this as minimal as possible and with a comment what generated it